themes/aframe/source/css/secondary.styl

```diff
@@ -62,6 +62,13 @@ a
     &:last-child .nav-link
         padding-bottom 0
 
+.subnav-label
+    font-size 11px
+    font-weight 600
+    line-height 1.2
+    padding 5px 0 10px
```

Comment:
this is a copy+paste of `.section-title` but with `font-weight: 600` - can you just turn `<span class="subnav-label">` to `<b class="subnav-label">`

and change `.subnav-title` selector to `.subnav-title, .subnav-label` (and we can make it DRY later).

also, I feel like `.subnav-label` needs a `margin-right: 5px;`-ish.